src/main/java/me/drendov/HouseKeeping/HouseKeeping.java

```diff
@@ -211,6 +211,10 @@ private void checkDependency(String name) {
         }
     }
 
+    public boolean checkWorld(String world) {
+        return this.getServer().getWorld(world) != null;
+    }
```

Comment:
`checkWorld` is implemented but never used

Reply:
oh yeah, that's another story) removed for now